apps/console/src/extensions/i18n/models/extensions.ts

```diff
@@ -2532,6 +2533,9 @@ export interface Extensions {
                 queryButton: {
                     label: string;
                 };
+                downloadButton : {
```

Comment:
```suggestion
                downloadButton: {
```
Check other places as well.

Reply:
Fixed with dd326a4fcaabec77fd74c72f6b9c3a4346203585